frontend/src/components/ProjectSelectionComponent.vue

```diff
@@ -1,14 +1,12 @@
 <template>
-  <div>
-    <b-form-select
-      v-model="selected"
-      class="form-select"
-      :options="getAllProjects"
-      @change="getUserStories"
-    ></b-form-select>
+  <div class="Test ?">
+    <div class="test123">
+      <autocomplete v-model="selected" :items="getProjectNames" @input="getUserStories" />
```

Comment:
It could make sense to place a placeholder in the input field.
> <autocomplete v-model="selected" :items="getProjectNames" @input="getUserStories" placeholder="Search projects..." />

Reply:
Do not forget i18n for placeholder.